app/views/caseload/new.html.erb

```diff
@@ -46,30 +54,31 @@
         </a>
         <%= sort_arrow('pom_responsibility') %>
       </th>
-      <th class="govuk-table__header" scope="col">Action</th>
     </tr>
     </thead>
     <tbody class="govuk-table__body">
     <% @new_cases.each_with_index do |allocation, i| %>
       <tr class="govuk-table__row offender_row_<%= i %>">
         <td aria-label="Prisoner name" class="govuk-table__cell">
           <span class='prisoner-name'>
-            <%= allocation.full_name %>
+            <%= link_to(highlight(allocation.full_name, @q), prison_prisoner_path(@prison.code, allocation.nomis_offender_id), class: "govuk-link" ) %>
```

Comment:
the 'highlight' call is superflous here IMHO because there is no search facility on the 'new-cases' screen